assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -0,0 +1,532 @@
+			data.rows = rows;
+			data.rowCount = rows.length;
+
+			// We pretend that the first row contains minimums and the last one maximums because we don't
+			// really need mathematically correct values and can simplify the process of finding this information.
+			data.minimums = [
+				{
+					dimensionValues: dimensions.map( ( dimension ) => {
+						if ( dimension === 'dateRange' ) {
+							return { value: 'date_range_0' };
+						}
+
+						return {
+							value: 'RESERVED_MIN',
+						};
+					} ),
+					metricValues: [ ...( rows[ 0 ]?.metricValues || [] ) ],
+				},
+			].concat(
+				hasDateRange
+					? [
+							{
+								dimensionValues: dimensions.map(
+									( dimension ) => {
+										if ( dimension === 'dateRange' ) {
+											return {
+												value: 'date_range_1',
+											};
+										}
+
+										return {
+											value: 'RESERVED_MIN',
+										};
+									}
+								),
+								metricValues: [
+									...( rows[ 0 ]?.metricValues || [] ),
+								],
+							},
+					  ]
+					: []
+			);
+
+			data.maximums = [
+				{
+					dimensionValues: dimensions.map( ( dimension ) => {
+						if ( dimension === 'dateRange' ) {
+							return { value: 'date_range_0' };
+						}
+
+						return {
+							value: 'RESERVED_MAX',
+						};
+					} ),
+					metricValues: [
+						...( rows[ rows.length - 1 ]?.metricValues || [] ),
+					],
+				},
+			].concat(
+				hasDateRange
+					? [
+							{
+								dimensionValues: dimensions.map(
+									( dimension ) => {
+										if ( dimension === 'dateRange' ) {
+											return {
+												value: 'date_range_1',
+											};
+										}
+
+										return {
+											value: 'RESERVED_MAX',
+										};
+									}
+								),
+								metricValues: [
+									...( rows[ rows.length - 1 ]
+										?.metricValues || [] ),
+								],
+							},
+					  ]
+					: []
+			);
+
+			// Same here, we pretend that the last row contains totals because we don't need it to be mathematically valid.
+			data.totals = [
+				{
+					dimensionValues: dimensions.map( ( dimension ) => {
+						if ( dimension === 'dateRange' ) {
+							return { value: 'date_range_0' };
+						}
+
+						return {
+							value: 'RESERVED_TOTAL',
+						};
+					} ),
+					metricValues: [
+						...( rows[ rows.length - 1 ]?.metricValues || [] ),
+					],
+				},
+			].concat(
+				hasDateRange
+					? [
+							{
+								dimensionValues: dimensions.map(
+									( dimension ) => {
+										if ( dimension === 'dateRange' ) {
+											return {
+												value: 'date_range_1',
+											};
+										}
+
+										return {
+											value: 'RESERVED_TOTAL',
+										};
+									}
+								),
+								metricValues: [
+									...( rows[ rows.length - 1 ]
+										?.metricValues || [] ),
+								],
+							},
+					  ]
+					: []
+			);
+		} );
+
+	// Set the original seed value for the faker.
+	faker.seed( originalSeedValue );
+
+	return [
+		{
```

Comment:
Sorry for not spotting this earlier, but this needs to be changed to only return a single object, rather than array with a single report object inside. There's a bit more info [here](https://github.com/google/site-kit-wp/issues/6172#issuecomment-1375638193) under "Single report".

This will obviously have a knock-on effect to the tests.

```suggestion
	return {
```